ssp/handler.go

```diff
@@ -25,55 +26,61 @@ func (s *Server) Handler() http.Handler {
 	return r
 }
 
-func (server *Server) NutHandler() http.HandlerFunc {
```

Comment:
I'd prefer we kept the existing interface for these handlers (a method returning the `http.HandlerFunc`). It gives the library a lot more flexibility as a consumer doesn't necessarily have to use the router we've chosen.

Reply:
Since the s.NutHandler is public and is a HandlerFunc already, a caller who has a Server instance can inject this handler in the same way as done in the Handler() method.

Wrapping in a closure adds an extra level of indirection. The only reason you would need to do that is to capture some local scope (which this method doesn't).

Reply:
You're absolutely right - my mistake. I do still like that there would be more consistency with the other `*Handler` methods, but I'm happy with this change.

Reply:
I thought about it :) 
I thought my first PR should be limited in scope and since those functions were in a different file I left them alone. I can open a new PR once this one lands.